src/Context/authContext.jsx

```diff
@@ -0,0 +1,28 @@
+import {createContext, useContext, useState}  from 'react';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({children}) => {
+
+    const getToken = localStorage.getItem("login-token");
+    const getLocalStorageUser = localStorage.getItem("user");
+    const [token, setToken] = useState(getToken || "")
+    const [user, setUser] = useState(JSON.parse(getLocalStorageUser) || "")
```

Comment:
Same here, you can put it ``` const getLocalStorageUser = localStorage.getItem("user")``` in  useState.
A separate utility function would be better for these cases.
Ex:
```
const getLocalStorageData=(key="")=>{
   return JSON.parse(localStorage.getItem(key))
}

const [user, setUser] = useState(getLocalStorageData("user")|| "")
```

You can use the utility function in the previous case also.